os/mm/mm_heap/mm_sem.c

```diff
@@ -106,6 +106,11 @@ void mm_seminitialize(FAR struct mm_heap_s *heap)
 	 * private data sets.
 	 */
 
+	if (heap == NULL) {
+		msemdbg("Fail to get heap.\n");
```

Comment:
We should say invalid input. This message could give wrong information.
"Invalid Heap address given, Fail to initialize sem"